content/references/http-websocket-apis/public-api-methods/ledger-methods/ledger_entry.md

```diff
@@ -557,10 +560,172 @@ rippled json ledger_entry '{ "ticket": { "owner": "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYf
 
 <!-- MULTICODE_BLOCK_END -->
 
+
 <!-- TODO: enable if/when Tickets are available on Mainnet
 [Try it! >](websocket-api-tool.html#ledger_entry-ticket)
 -->
 
+### Get Bridge Object
+
+Retrieve a [Bridge object](bridge.html) which connects a locking chain and an issuing chain. Can be provided as string (object ID of the Bridge) or as an object. _(Added by the [Sidechains amendment][])_
+
+| `Field`                 | Type                       | Description           |
+|:------------------------|:---------------------------|:----------------------|
+| `bridge`                | Object or String           | The [[Bridge object](bridge.html) to retrieve. |
+| ``          | String - [Address][]       | _(Required if `ticket` is specified as an object)_ The owner of the Ticket object. |
+
+<!-- MULTICODE_BLOCK_START -->
+
+*WebSocket*
+
+```json
+{
+  "id": "example_get_bridge",
+  "command": "ledger_entry",
+  "bridge": {
+    "issuing_chain_door": "rKeSSvHvaMZJp9ykaxutVwkhZgWuWMLnQt",
+    "issuing_chain_issue": "XRP",
+    "locking_chain_door": "rJvExveLEL4jNDEeLKCVdxaSCN9cEBnEQC",
+    "locking_chain_issue": "XRP"
+  },
+  "ledger_index": "validated"
+}
+```
+
+*JSON-RPC*
+
+```json
+{
+  "method": "ledger_entry",
+  "params": [{
+   "bridge": {
+    "issuing_chain_door": "rKeSSvHvaMZJp9ykaxutVwkhZgWuWMLnQt",
+    "issuing_chain_issue": "XRP",
+    "locking_chain_door": "rJvExveLEL4jNDEeLKCVdxaSCN9cEBnEQC",
+    "locking_chain_issue": "XRP"
+   },
+  "ledger_index": "validated"
+  }]
+}
+```
+
+*Commandline*
+
+```sh
+rippled json ledger_entry '{ "bridge": { "issuing_chain_door": "rKeSSvHvaMZJp9ykaxutVwkhZgWuWMLnQt", "issuing_chain_issue": "XRP", "locking_chain_door": "rJvExveLEL4jNDEeLKCVdxaSCN9cEBnEQC", "locking_chain_issue": "XRP" }, "ledger_index": "validated" }'
+```
+
+<!-- MULTICODE_BLOCK_END -->
+
+### Get xchain_claim_id
+
+Retrieve a [Crosschain Claim ID object](xchains.html). Can be provided as string (object ID of the crosschain claim) or as an object. _(Added by the [Sidechains amendment][])_
```

Comment:
Not sure what page `xchains.html` is supposed to be, but it doesn't exist yet in this PR, so this is a broken link.